src/content/blog/val-town-newsletter-22.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: Val Town Newsletter 22
+description: Townie upgrades, Scoped API permissions, Fal partnership
+pubDate: November 27, 2024
+author: Steve Krouse
+---
+
+<style>
+  {`
+    .lineup {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      grid-gap: 20px;
+    }
+    figcaption {
+      margin-top: -20px;
+    }
+    .border {
+      border: 2px solid #eaeaea;
+      padding: 5px;
+    }
+    #subscribe {
+      font-size: 20px; 
+      border: 2px solid #143f9d; 
+      text-align:center; 
+      border-radius: 5px; 
+      text-decoration: none; 
+      display: block;
+    }
+    #subscribe:hover {
+      background-color: #143f9d;
+      color: white;
+    }
+`}
+</style>
+
+import { Image } from "astro:assets";
+import Demo from "./fal/demo.png";
+import Faltown from "./fal/faltown.png";
+import Pricing from "./val-town-newsletter-22/Pricing.png";
+import SettingsBlob from "./val-town-newsletter-22/SettingsBlob.png";
+import BlobAdmin from "./val-town-newsletter-22/BlobAdmin.png";
+
+Val Town is growing up! We're working on new features and infra that will fundamentally unlock the scope of what you can build in Val Town. We have two major efforts running in parallel: projects and scale.
+
+_Projects_ are containers for multiple vals, folders, and files – enabling you to create infinitely complex real-world apps – instead of being limited to a single long val. Internal to a project, vals will use relative imports (ie `./foo/bar`), so they can be forked as a unit, and merged back in as a unit. Projects will also improve Townie's ability to help write your vals in more modular files – so it doesn't have to generate and re-generate all the code on every change.
```

Comment:
```suggestion
_Projects_ are containers for multiple vals, folders, and files. The enable you to create infinitely complex real-world apps, instead of being limited to a single long val. Internal to a project, vals will use relative imports (ie `./foo/bar`), so they can be forked as a unit, and merged back in as a unit. Projects will also improve Townie's ability to help write your vals in more modular files, so that it doesn't have to regenerate all the code on every change.
```